libs/portals/shared-modules/delegations/src/lib/messages.ts

```diff
@@ -209,4 +209,8 @@ export const m = defineMessages({
     id: 'sp.access-control-delegations:access-valid-to',
     defaultMessage: 'Í gildi til',
   },
+  accessCreationSuccess: {
+    id: 'sp.access-control-delegations:access-creation-success',
```

Comment:
Has this key been uploaded to Contentful and the corresponding EN translation added?